### PR TITLE
(2894) Document activities download for annual impact metrics

### PR DESCRIPTION
**Pull request by jkempster34**
## Changes in this PR
Document activities download for annual impact metrics

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [ ] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by jkempster34** [comment]
[Document any common support tasks](https://trello.com/c/FL7UdL2W/2894-document-any-common-support-tasks)

**Review by yndajas — APPROVED** [review]
Makes sense to me

File: doc/support_tasks/download_activities_for_annual_impact_metrics.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Every July, support will need to generate a CSV of activities that will be used
+to pre-populate an annual collaborative spreadsheet of fund impact metrics.
+
+1. Connect to the production environment using SSH. See [console
+   access](/doc/console-access.md).
+   ```shell
+   cf ssh beis-roda-prod
+   ```
+2. Run the annual fund impact metrics Rails task. The resulting CSV will be
```
Comment (shuldt) [inline doc/support_tasks/download_activities_for_annual_impact_metrics.md:11]: potentially add a path to the file here?

**Review by shuldt — APPROVED** [review]
lgtm

**Pull request merged**